radio/src/targets/sky9x/rtc_driver.cpp

```diff
@@ -280,3 +280,25 @@ void rtcInit()
   coprocReadData();
 #endif
 }
+

```

Comment:
Yes, as far as I could see, the 2 and 4 flags for `Rtc_write_pending` are not (yet) used anywhere.